docs/scripts/uploadBenchmark.py

```diff
@@ -104,18 +108,29 @@ def create_folder_structure(
     print(f"total_size_of_files_bytes: {total_size_of_files_bytes}")
     print(f"size_of_each_file_bytes: {size_of_each_file_bytes}")
 
+    if total_size_of_files_mib >= 1024 * 5:
+        print(
+            f"total_size_of_files_gib: {total_size_of_files_bytes / (1024 * MiB)}"
+        )
+        # 32 MiB chunks
```

Comment:
since the files that I want to upload are huge, I think it no longer makes sense to upload them as 1 MiB chunk. I changed it to 32 MiB chunks for files that are bigger than 5GB. I am not sure if that affects anything.